src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const styles = sourceString
+    .split(';')
+    .filter((style) => style.trim().length > 0);
+  const styleOBJ = {};
+
+  styles.forEach((style) => {
+    const [key, value] = style.split(':').map((part) => part.trim());
```

Comment:
When splitting the style string, consider checking for the presence of both key and value to avoid adding undefined properties to the object in case of malformed style strings.